src/app/signin/page.css

```diff
@@ -163,13 +144,13 @@ h2 {
   font-weight: bold;
 }
 
-.signup-link {
+.sign-link {
   text-decoration: none;
   font-weight: bold;
   color: white;
   transition: all ease-in-out 200ms;
```

Comment:
I would make this link accessible. I know that you did same as in the design, but currently it is not clear that Sign In/Sign Up links are clickable. I would change to something like this: ```
.sign-link {
    text-decoration: underline;
    color: mediumseagreen;
    transition: all .2s ease-in-out;
}
```
What do you think?

Reply:
Thanks for the suggestion,
At first my approach was different but Vitalina suggested this solution

Sure, I agree with you.
I will try your suggestion.

Thanks